src/Digdir.Domain.Dialogporten.Application/Features/V1/ServiceOwner/Dialogs/Commands/Create/CreateDialogCommand.cs

```diff
@@ -128,6 +128,13 @@ public async Task<CreateDialogResult> Handle(CreateDialogCommand request, Cancel
             _domainContext.AddError(DomainFailure.EntityExists<DialogTransmission>(existingTransmissionIds));
         }
 
+        var transmissionAttachements = dialog.Transmissions.SelectMany(x => x.Attachments);
```

Comment:
transmissionAttachments*

Reply:
We are checking this on create because we want to validate that no other attachments (unrelated to this dialog) might share the same id?